src/sentry_settings.h

```diff
@@ -0,0 +1,43 @@
+#ifndef SENTRY_SETTINGS_H
+#define SENTRY_SETTINGS_H
+
+#include <godot_cpp/core/object.hpp>
+#include <godot_cpp/core/property_info.hpp>
+#include <godot_cpp/variant/char_string.hpp>
+#include <godot_cpp/variant/string.hpp>
+#include <godot_cpp/variant/variant.hpp>
+
+class SentrySettings {
```

Comment:
I called it Settings btw, because it's an interface to `ProjectSettings` in Godot. It's a singleton we currently use internally to access those options:
![image](https://github.com/user-attachments/assets/ab7c0b65-a714-4d07-ab40-2d49727d6ea1)

Reply:
Addressed in 321018a